packages/admin/admin-rte/src/core/translation/htmlToState.ts

```diff
@@ -0,0 +1,42 @@
+import { convertFromRaw, convertToRaw, EditorState } from "draft-js";
+import { stateFromHTML } from "draft-js-import-html";
+
+export function htmlToState({
+    html,
+    linkDataList,
+}: {
+    html: string;
+    linkDataList: {
+        id: string;
+        data: any;
+    }[];
+}) {
+    const translatedContentState = stateFromHTML(html, {
+        customInlineFn: (element, { Style, Entity }) => {
```

Comment:
What about other inline styles such as italic and bold? Do they work out-of-the-block?